cookbooks/cdo-apps/metadata.rb

```diff
@@ -25,7 +25,6 @@
 depends 'chef_hostname'
 depends 'poise-service'
 depends 'cdo-redis'
-depends 'cdo-solr'
```

Comment:
I always forget - do we have to bump the `cdo-apps` version, or does that happen automagically now?

Reply:
@wjordan Do you know?